nibabies/workflows/bold/alignment.py

```diff
@@ -0,0 +1,268 @@
+"""
+Subcortical alignment into MNI space
+"""
+
+from nibabies.interfaces.nibabel import MergeROIs
+
+
+def init_subcortical_mni_alignment_wf(*, repetition_time, name='subcortical_mni_alignment_wf'):
+    """
+    Align individual subcortical structures into MNI space.
+
+    This is a nipype workflow port of the DCAN infant pipeline.
+    https://github.com/DCAN-Labs/dcan-infant-pipeline/ \
+    blob/247e19/fMRISurface/scripts/SubcorticalAlign_ROIs.sh
+
+
+    Parameters
+    ----------
+    repetition_time : :obj:`int`
+        BOLD file's TR
+    name : :obj:`str`
+        Name of the workflow
+
+    Inputs
+    ------
+    bold_file : :obj:`str`
+        BOLD file
+    bold_roi : :obj:`str`
+        File containing ROIs in BOLD space
+    atlas_roi : :obj:`str`
+        File containing ROIs in atlas space
+    std_xfm : :obj:`str`
+        File containing transform to the standard (MNI) space
+
+    Outputs
+    -------
+    subcortical_file : :obj:`str`
+        The BOLD file in atlas space with each ROI individually aligned.
+    """
+    from nipype.pipeline import engine as pe
+    from nipype.interfaces import utility as niu, fsl
+    from ...interfaces.workbench import (
+        CiftiCreateDenseTimeseries,
+        CiftiCreateLabel,
+        CiftiDilate,
+        CiftiResample,
+        CiftiSeparate,
+        CiftiSmooth,
+        VolumeAffineResample,
+        VolumeAllLabelsToROIs,
+        VolumeLabelExportTable,
+        VolumeLabelImport,
+    )
+    from niworkflows.engine.workflows import LiterateWorkflow as Workflow
+
+    inputnode = pe.Node(
+        niu.IdentityInterface(fields=["bold_file", "bold_roi", "atlas_roi", "atlas_xfm"]),
+        name="inputnode",
+    )
+    outputnode = pe.Node(niu.IdentityInterface(fields=["subcortical_file"]), name='outputnode')
+
+    applyxfm_atlas = pe.Node(fsl.FLIRT(), name="applyxfm_atlas")
+    vol_resample = pe.Node(
+        VolumeAffineResample(method="ENCLOSING_VOXEL", flirt=True),
+        name="vol_resample"
+    )
```

Comment:
In the original, there's a [`CreateDenseTimeseries` here](https://github.com/DCAN-Labs/dcan-infant-pipeline/blob/master/fMRISurface/scripts/SubcorticalAlign_ROIs.sh#L60). What happened?

Reply:
That was only used as a template when joining all the ROIs into a CIFTI for the previous implementation, see https://github.com/nipreps/nibabies/pull/72/commits/3d909abd6edf89628248358148dd08e9b8aec24b